src/main/resources/com/google/api/codegen/py/docs/conf.py.snip

```diff
@@ -123,12 +128,20 @@
   
   @# The theme to use for HTML and HTML Help pages.  See the documentation for
   @# a list of builtin themes.
-  html_theme = 'sphinx_rtd_theme'
+  html_theme = 'alabaster'
   
   @# Theme options are theme-specific and customize the look and feel of a theme
   @# further.  For a list of options available for each theme, see the
   @# documentation.
-  @#html_theme_options = {}
+  html_theme_options = {
+      @#'description': 'Google Cloud Client Libraries for Python',
+      @#'github_user': 'GoogleCloudPlatform',
+      @#'github_repo': 'google-cloud-python',
+      @#'github_banner': True,
```

Comment:
Seems like we could omit those four lines:  some are stale, and they get dropped anyway.

Reply:
It looks like these options are used in the top-level conf.py, so I re-added them. The fields will need to be modified post repo split.